ssl/ssl_conf.c

```diff
@@ -520,6 +520,22 @@ static int cmd_DHParameters(SSL_CONF_CTX *cctx, const char *value)
     return rv > 0;
 }
 #endif
+
+static int cmd_RecordPadding(SSL_CONF_CTX *cctx, const char *value)
+{
+    size_t block_size = (size_t)atoi(value);
```

Comment:
Is there a risk that `value` could be -ve (which would get cast to a positive). Perhaps we should sanity check the value *before* the cast?